tests/e2e/install_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = When("Installing Koperator", func() {
+	var kubeconfigPath string
```

Comment:
I think we can use global variable for kubeconfigPath and kubecontextName. In the BeforeSuite there is a check and value (https://github.com/banzaicloud/koperator/blob/3f98e5865cd2eefaa3872e86ed1c6f60b646447d/tests/e2e/koperator_suite_test.go#L39)

Reply:
I opted not to use a global variable for those, because we would want to pass it in as a parameter once the multiple K8s cluster support is done.
Imagine a scenario where we want to run this scenario on AKS, AWS, GKE, RHOS, we would run the same scenario with different clusters as parameters.
I didn't introduce these as parameters yet, because I didn't have time to check what would be the best to pass those parameters into such Ginkgo scenarios yet.

Also globals are anti-pattern, because they are not properly access-bound so I'm staying away from those when it's possible without significant costs.

Reply:
I think it is OK to keep it as it is. Later probably we need to add k8s health test like here: https://github.com/banzaicloud/koperator/blob/f7fe06f20be0d303c28575169aaaa0ef1f0d1888/tests/e2e/koperator_suite_test.go#L43 also into tests to check those other clusters health